compiler/rustc_const_eval/src/interpret/projection.rs

```diff
@@ -149,6 +149,7 @@ where
                 None => {
                     // For unsized types with an extern type tail we perform no adjustments.
                     // NOTE: keep this in sync with `PlaceRef::project_field` in the codegen backend.
+                    // FIXME: that one? compiler/rustc_codegen_ssa/src/mir/place.rs
```

Comment:
Specifically this:

https://github.com/rust-lang/rust/blob/79bfd93d5a9cf73d312b72158c1d2289453c18c7/compiler/rustc_codegen_ssa/src/mir/place.rs#L148-L154

I don't see how an automatic checker can help here.

Reply:
I just run over codebase, found few comments like "keep in sync with" and wrapped them in tags. Of course this check can't say if two places is valid, only ping author to give it chance to check things manually.